shared/common-adapters/user-proofs.native.js

```diff
@@ -31,7 +32,7 @@ function MissingProofRow ({missingProof, style}: {missingProof: MissingProof, st
           </Box>
         </Box>
         <Box style={styleStatusIconContainer}>
-          <Icon type={'iconfont-proof-placeholder'} style={{...styleStatusIcon('iconfont-proof-placeholder'), color: missingColor}} />
+          <Icon type={'iconfont-proof-placeholder'} style={{...styleStatusIcon('iconfont-proof-placeholder'), color: globalColors.lightGrey}} />
```

Comment:
Could change `missingColor = globalColors.lightGrey` above if you want to make this change for all "missingColor". If this is specific to just this place is OK though and nm.